template/template_test.go

```diff
@@ -21,8 +21,8 @@ import (
 )
 
 const (
-	OpenshiftIOAPI     = "http://api.openshift.io"
-	OpenShiftIOAuthAPI = "https://auth.openshift.io"
+	OpenshiftIOAPI     = "http://api.prod-preview.openshift.io"
```

Comment:
why do you need to change this?

Reply:
The same problem as in tests. Prod accounts may be deactivated (as they did for accounts used here before that PR).

Reply:
ok